libraries/AdaptiveExpressions/BuiltinFunctions/ConvertFromUtc.cs

```diff
@@ -23,24 +25,30 @@ private static (object value, string error) Evaluator(Expression expression, IMe
             object value = null;
             string error = null;
             IReadOnlyList<object> args;
+            var locale = options.Locale != null ? new CultureInfo(options.Locale) : Thread.CurrentThread.CurrentCulture;
+            var format = FunctionUtils.DefaultDateTimeFormat;
```

Comment:
Doesn't options.locale always have a locale of CurrentCulture by construction?  It would only be null if someone explicitly set it to null.